questions/not_enough_skills_to_deliver_a_done_increment.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+title: "Not enough skills to deliver a Done increment"
+parent: Katas
+---
+Developers discuss that due to the absence of a Developer it no longer has all the skills required to deliver a working "Done" increment according to the definitions of "Done". It wants to adjust the definitions of "Done".
```

Comment:
"according to the Definitions of Done." -> "according to their Definition of Done (DoD)."
1. "their" instead of "the" 
2. Definition instead of Definitions
3. Add abbreviation as we use it in the next sentence?

Reply:
Fixed the suggestion